src/tests/topp/TextExporter_4_output_proteins.txt

```diff
@@ -1,7 +1,7 @@
 #RUN	run_id	score_type	score_direction	date_time	search_engine_version	parameters
-#PROTEIN	score	rank	accession	coverage	sequence
+#PROTEIN	score	rank	accession	protein_name	coverage	sequence
 RUN	Mascot_2006-01-12T12:13:14	MOWSE	higher-score-better	2006-01-12T12:13:14	2.1.0	db=MSDB, db_version=1.0, taxonomy=, charges=+1, +2, mass_type=average, fixed_modifications=, variable_modifications=, enzyme=Trypsin, missed_cleavages=0, peak_mass_tolerance=0.3, precursor_mass_tolerance=1
-PROTEIN	34.4000015258789	0	PROT1	42.8571428571429	ABCDEFG
-PROTEIN	24.3999996185303	0	PROT2	37.5	DEFGHIGK
+PROTEIN	34.4000015258789	0	PROT1		42.8571428571429	ABCDEFG
+PROTEIN	24.3999996185303	0	PROT2		37.5	DEFGHIGK
```

Comment:
now it seems in all TOPP tests the `protein_name` is just empty. Can you write a test that would also test that the protein name is written out correctly?